module/VuFind/src/VuFind/Controller/LibraryCardsController.php

```diff
@@ -54,29 +54,6 @@ public function homeAction()
             return $this->forceLogin();
         }
 
-        // Check for "delete card" request; parameter may be in GET or POST depending
-        // on calling context.
-        $deleteId = $this->params()->fromPost(
```

Comment:
Similarly, it is not clear to me why this logic is being removed; I believe it is actually used.

Reply:
I don't think it is used - delete library card is its own controller action. Two methods `performDeleteLibraryCard` and `confirmDeleteLibraryCard` used in code below are not defined anywhere.

Reply:
Ahh, you're right -- I had forgotten that this got moved to a separate controller.